frontend/tests/unit-tests/EnsembleParameters.test.ts

```diff
@@ -0,0 +1,118 @@
+        }
+    });
+
+    test("Check for parameter existence", () => {
+        const ensParams = new EnsembleParameters(PARAM_ARR);
+
+        expect(ensParams.hasParameter(ParameterIdent.fromNameAndGroup("cparam_10", null))).toBe(true);
+        expect(ensParams.hasParameter(ParameterIdent.fromNameAndGroup("cparam_50", "grp1"))).toBe(true);
+        expect(ensParams.hasParameter(ParameterIdent.fromNameAndGroup("cparam_50", "grp2"))).toBe(true);
+
+        expect(ensParams.hasParameter(ParameterIdent.fromNameAndGroup("aName", "aGroup"))).toBe(false);
+        expect(ensParams.hasParameter(ParameterIdent.fromNameAndGroup("", ""))).toBe(false);
+        expect(ensParams.hasParameter(ParameterIdent.fromNameAndGroup("cparam_10", ""))).toBe(false);
+        expect(ensParams.hasParameter(ParameterIdent.fromNameAndGroup("cparam_50", null))).toBe(false);
+    });
+
+    test("Get parameters", () => {
+        const ensParams = new EnsembleParameters(PARAM_ARR);
+        {
+            const par = ensParams.getParameter(ParameterIdent.fromNameAndGroup("cparam_10", null));
+            expect(par.type).toEqual(ParameterType.CONTINUOUS);
+            expect(par.name).toEqual("cparam_10");
+            expect(par.groupName).toEqual(null);
+            expect(par.values).toEqual([11, 12, 19]);
+        }
+        {
+            const par = ensParams.getParameter(ParameterIdent.fromNameAndGroup("cparam_50", "grp2"));
+            expect(par.type).toEqual(ParameterType.CONTINUOUS);
+            expect(par.name).toEqual("cparam_50");
+            expect(par.groupName).toEqual("grp2");
+            expect(par.values).toEqual([55, 56, 59]);
+        }
+        {
+            const par = ensParams.getParameter(ParameterIdent.fromNameAndGroup("dparam_B", null));
+            expect(par.type).toEqual(ParameterType.DISCRETE);
+            expect(par.name).toEqual("dparam_B");
+            expect(par.groupName).toEqual(null);
+            expect(par.values).toEqual(["A", "B", "C"]);
+        }
+    });
+
+    test("Check that getting non-existing parameter throws", () => {
+        const ensParams = new EnsembleParameters(PARAM_ARR);
+        expect(() => ensParams.getParameter(ParameterIdent.fromNameAndGroup("someBogusName", null))).toThrow();
+    });
+
+    test("Test getting min/max values for continuous parameter", () => {
+        const ensParams = new EnsembleParameters(PARAM_ARR);
+        {
+            const minMax = ensParams.getContinuousParameterMinMax(ParameterIdent.fromNameAndGroup("cparam_10", null));
+            expect(minMax).toEqual(new MinMax(11, 19));
+        }
+        {
+            const minMax = ensParams.getContinuousParameterMinMax(ParameterIdent.fromNameAndGroup("cparam_50", "grp1"));
+            expect(minMax).toEqual(new MinMax(51, 54));
+        }
+    });
+});
+
+
+describe("ParameterIdent tests", () => {
+    test("Conversion to/from string", () => {
```

Comment:
Perhaps a matter of taste, but I like to test single functions per test, using hard coded string as expected result for string conversion and hard coded strings as argument for generators from string (`fromString()`method). I.e. test `fromNameAndGroup("aName", "aGroup")`-method towards a `new ParameterIdent("aName", "aGroup")`-object and its public properties? Then test `toString()`-method towards a hard coded expected string? Then testing `fromString()` with the hardcoded string?

Reply:
Will make some adjustments to the test